File: todoapp/lib/models/todo.dart
class Todo {
  late String id;
  late String title;
  late bool status;

  Todo({required this.title, required this.status, required this.id});

  factory Todo.fromJson(Map<String, dynamic> json) {
    return Todo(
      id: json['_id'],
      title: json['title'],
      status: json['status'],
    );
  }
}

File: backend/models/todo.js
const mongoose = require("mongoose");
const Schema = mongoose.Schema;

const todoSchema = new Schema({
    title : {
        type : String,
        required : true,
    },
    status : {
        type : Boolean,
        required : true,
    }, 
});

const Todo = mongoose.model('Todo',todoSchema);

module.exports = Todo;